docs/guide/index.md

```diff
@@ -18,70 +22,110 @@ A-Frame is an open-source framework for creating 3D and virtual reality experien
-                position="-1 0.5 1" rotation="0 45 0"
-                physics="boundingBox: 1 1 1; mass: 2"
-                explode="on: collide"
-                template="src: butterflies.template; type: nunjucks"
-                speech-controls="trigger: siri"></a-entity>
-
-      <a-entity collada-model="monster.dae"></a-entity>
-
-      <a-entity geometry="primitive: plane" material="src: url(fox.png)"></a-entity>
-
-      <a-entity geometry="primitive: sphere; radius: 5000"
-                material="color: #EF2D5E"
-                scale="-1 1 1"></a-entity>
-    </a-scene>
-  </body>
-</html>
+<a-entity geometry="primitive: box; depth: 2"
+          material="color: #6173F4; opacity: 0.8"
+          physics="mass: 5; boundingBox: 1 1 2"
+          explode="on: physics-collide; intensity: 3"></a-entity>
 ```
 
-Read through the documentation front-to-back for more details, and if you have any questions, join the other hundreds of developers on [Slack][slack]! Welcome to the future of the web.
+## Why A-Frame?
+
+A-Frame was built by the [Mozilla VR team][mozvr] to make it **quicker** and
+**easier** to build 3D/VR scenes in order for them to prototype faster and to
+bridge the web development community into the WebVR ecosystem. For WebVR to
+succeed, it needs content.  There are only a handful of WebGL developers in the
+world, but there are *millions* of web developers, designers, and artists.
+A-Frame puts 3D/VR content creation into the hands of everyone.
+
+### A-Frame Reduces Boilerplate
+
+Without A-Frame, starting a proper WebVR project is a lot of effort. You need
+to know what you are doing and repeat the same tedious work for every scene:
+
+Import the WebVR polyfill, look through the three.js repository examples for
+`VREffect`, declare a canvas, create a camera, set up lights, instantiate a
+render loop, wire everything together, build UI to enter VR, etc.
+
+Would toy sites like [zombo.com](https://zombo.com) exist if simply getting
+started required this much work? WebVR should thrive with long tail bite-sized
+experiences, but boilerplate is a strong barrier to motivation.
+
+In A-Frame, all boilerplate is reduced to one line of HTML: **`<a-scene>`**
+
+<video autoplay loop src="/videos/boilerplate.mp4"></video>
+
+All pieces of work in general are greatly reduced.  Rather than creating a
+mesh, creating a geometry, creating a material, and appending to scene...we can
+just add one of HTML.
+
+### A-Frame is Tailored for Web Developers
+
+HTML is arguably the easiest interface in all of computing, and it is
+universally understood amongst web developers and even non-developers. With
+A-Frame based on the DOM, we can manipulate scenes like we would with other web
+application: `getAttribute`, `setAttribute`, `querySelector`, etc., it's all
+there.
+
+This also means that most JavaScript frameworks and libraries integrate with
+A-Frame out-of-the-box. d3, React, Vue.js, Meteor, jQuery all work like a
```

Comment:
it would be nice to link to the libraries or [this section of awesome-aframe](https://github.com/aframevr/awesome-aframe#integration)